src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.BlockProductionTransactionPicker.cs

```diff
@@ -4,24 +4,29 @@
 using System;
 using System.Collections.Generic;
 using Nethermind.Core;
+using Nethermind.Core.Extensions;
 using Nethermind.Core.Specs;
 using Nethermind.Evm;
 using Nethermind.Evm.TransactionProcessing;
 using Nethermind.Int256;
 using Nethermind.State;
+using Nethermind.TxPool;
 
 namespace Nethermind.Consensus.Processing
 {
     public partial class BlockProcessor
     {
         public class BlockProductionTransactionPicker : IBlockProductionTransactionPicker
         {
+            private readonly long _maxTxLengthBytes;
+
             protected readonly ISpecProvider _specProvider;
             private readonly bool _ignoreEip3607;
 
-            public BlockProductionTransactionPicker(ISpecProvider specProvider, bool ignoreEip3607 = false)
+            public BlockProductionTransactionPicker(ISpecProvider specProvider, long maxTxLengthKilobytes = 9728, bool ignoreEip3607 = false)
```

Comment:
nullable value in in constructor parameters and set it to const value inside? shared with the config value?

Reply:
or pass config in constructor?

Reply:
or use same named const in both places

Reply:
Used const